### PR TITLE
Examples for Arduino mega2560

### PR DESCRIPTION
**Pull request by explicite**
- Examples like in https://github.com/Rahix/avr-hal/commit/8f6928cbfd2605bc0ea9e812a66456a782108043
- Renames for ports file

### CODE REVIEW

**Comment by explicite** [comment]
@Rahix I was able to test this two examples. Can you please review?

File: boards/arduino-mega2560/mega2560-runner.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+printf "   .text   %s (exact: %d)\n" "$(numfmt --to=si --padding=9 "$SIZE_TEXT")" "$SIZE_TEXT"
+printf "   .data   %s (exact: %d)\n" "$(numfmt --to=si --padding=9 "$SIZE_DATA")" "$SIZE_DATA"
+printf "   .bss    %s (exact: %d)\n" "$(numfmt --to=si --padding=9 "$SIZE_BSS")" "$SIZE_BSS"
+printf "\n"
+printf "Please bring up the bootloader and press ENTER!\n"
+read -r
+printf "Attempting to flash ...\n"PE0
+printf "\n"
+
+avrdude -qq -patmega2560 -cwiring -P"${SERIAL_PORT}" -b57600 -D "-Uflash:w:$1:e"
```
Comment (Rahix) [inline boards/arduino-mega2560/mega2560-runner.sh:60]: In `.cargo/config.toml` you have

```toml
[target.'cfg(target_arch = "avr")']
runner = "../../boards/arduino-mega2560/mega2560-runner.sh"
```

which means the old runner is used.  I would keep it like this for now which means you can delete this copy of the runner here.

Comment (explicite) [reply boards/arduino-mega2560/mega2560-runner.sh]: Done.

**Review by Rahix — APPROVED** [review]
Looks good, thanks!

**Pull request merged**